src/component/Flow.tsx

```diff
@@ -73,7 +73,7 @@ export const dragSelectSpecId = 'drag-select';
 
 export const getGhostUI = (ghostNode: FlowNode = {} as any) => ({
     position: GHOST_POSITION_INITIAL,
-    ...(ghostNode.router ? { type: 'wait_for_response' } : {})
+    ...(ghostNode.router ? { type: Types.wait_for_response } : {})
```

Comment:
Agree on using constants for these, but they sort of feel like they should be with flowTypes since they are describing the internals of the flow definition, but I see you have operators too. Those have a similar vibe to me, hmmm.. we can chat about that later I spose.

Reply:
Open to anything here, just wanted to ensure we type these before too long